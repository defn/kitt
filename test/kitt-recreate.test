#!/bin/bash

function main {
  kitt recreate --help
}

main "$@"
