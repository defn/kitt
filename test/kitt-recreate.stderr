USAGE: kitt recreate [flags] --  args

Calls docker-compose to remove containers and bring them up from scratch.
Use recreate when configuration changes since restart will retain the running
config.

args: arguments to docker-compose

flags:
  -h,--help:  show this help (default: false)
